projects/plugins/jetpack/extensions/shared/memberships/subscribers-affirmation.js

```diff
@@ -289,9 +303,22 @@ function SubscribersAffirmation( { accessLevel, prePublish = false } ) {
 		<p>
 			{ createInterpolateElement( text, {
 				strong: <strong />,
+				link: <a href={ getJetpackEmailStatsLink( blogId, postId ) } />,
 			} ) }
 		</p>
 	);
 }
 
+/**
+ * Get the Jetpack email stats link for the given post ID.
+ *
+ * @param {number} blogId - The ID of the blog.
+ * @param {number} postId - The ID of the post.
+ *
+ * @return {string} - The Jetpack email stats link for the given post.
+ */
+function getJetpackEmailStatsLink( blogId, postId ) {
+	return `/wp-admin/admin.php?page=stats#!/stats/email/opens/day/${ postId }/${ blogId }`;
```

Comment:
For simple sites we also have `wordpress.com/stats/email/opens/day/...` in Calypso. But given the untangling of Calypso screens going on, it seems fine to point from the editor to wp-admin.